controller.tf

```
@@ -58,22 +60,22 @@ data "external" "kubeadm_join" {
   program = ["./scripts/kubeadm-token.sh"]
```

Let's add this as suggested by @tabdulazim 
```suggestion
  program = ["${path.module}/scripts/kubeadm-token.sh"]
```

---

Added